util/network/src/client_version.rs

```diff
@@ -0,0 +1,516 @@
+// Copyright 2015-2019 Parity Technologies (UK) Ltd.
+// This file is part of Parity Ethereum.
+
+// Parity Ethereum is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity Ethereum is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity Ethereum.  If not, see <http://www.gnu.org/licenses/>.
+
+#![warn(missing_docs)]
+
+//! Parse ethereum client ID strings and provide querying functionality
+
+use semver::Version;
+use std::fmt;
+
+/// Parity client string prefix
+const LEGACY_CLIENT_ID_PREFIX: &str = "Parity";
+const PARITY_CLIENT_ID_PREFIX: &str = "Parity-Ethereum";
+
+const PARITY_ID_STRING_MINIMUM_LENGTH: usize = 4;
```

Comment:
I would suggest renaming this constant (`s/LENGTH/TOKENS` ?), since it's not clear w/o a context, that the length is in tokens, separated by `/`, and not bytes.
Also consider making it a local const in `fn parse_parity_format`, it's the only place where this constant is used.

Reply:
Ok, good points both.